src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10210,7 +10210,7 @@ namespace UnitarySystems {
         }
 
         // Determine the m_Staged status
-        if (allocated(state.dataZoneCtrls->StageZoneLogic) && this->m_DesignSpecMSHPIndex > 0) {
+        if (allocated(state.dataZoneCtrls->StageZoneLogic) && this->m_DesignSpecMSHPIndex > -1) {
```

Comment:
From a purely personal preference standpoint, I'd prefer ` >= 0 ` rather than ` > -1 `, but it's totally fine here.